packages/react-model-ui/package.json

```diff
@@ -35,12 +35,12 @@
     "@crossbreeze/protocol": "0.0.0",
     "@emotion/react": "^11.10.6",
     "@emotion/styled": "^11.10.6",
-    "@mui/icons-material": "^5.11.16",
-    "@mui/material": "^5.13.5",
-    "@mui/x-data-grid": "^6.2.1",
+    "@mui/icons-material": "^5.15.15",
+    "@mui/lab": "5.0.0-alpha.170",
```

Comment:
@martin-fleck-at I don't see this dependency being used anywhere.

Reply:
@harmen-xb Good catch! I forgot to remove it after a few rounds of iterations. I can do it as soon as your build runs through again.